app/scripts/transportapi.js

```diff
@@ -142,6 +142,20 @@
       return transportAPIcall(url, successFunction);
     },
 
+    busPerformance: function(codes, successFunction, timeout) {
+      var codesParam = codes.join(',');
+
+      var url = transportAPIbaseURL +
```

Comment:
Fine for this commit. But looks a fugly way of constructing a URL with query string parameters? JQuery has methods for this, but we don't want to have to bundle that lib. Any other lib we could reuse?